internal/characters/ganyu/burst.go

```diff
@@ -7,13 +7,15 @@ import (
 	"github.com/genshinsim/gcsim/pkg/core/combat"
 	"github.com/genshinsim/gcsim/pkg/core/glog"
 	"github.com/genshinsim/gcsim/pkg/core/player/character"
-	"github.com/genshinsim/gcsim/pkg/enemy"
 	"github.com/genshinsim/gcsim/pkg/modifier"
 )
 
 var burstFrames []int
 
-const burstStart = 130
+const (
+	burstStart   = 130
+	burstMarkKey = "ganyu-burst-mark"
+)
```

Comment:
That's because in those trials there were 2 enemies in the radius. So it's only 148 in those trials, because the first tick hit the enemy I wasn't looking at.